src/org/zaproxy/zap/extension/spider/SpiderAPI.java

```diff
@@ -105,16 +106,16 @@
 
 	/**
 	 * Instantiates a new spider API.
-	 * 
+	 *
 	 * @param extension the extension
 	 */
 	public SpiderAPI(ExtensionSpider extension) {
 		this.extension = extension;
 		// Register the actions
 		this.addApiAction(new ApiAction(ACTION_START_SCAN, new String[] { PARAM_URL },
-				new String[] { PARAM_MAX_CHILDREN }));
-		this.addApiAction(new ApiAction(ACTION_START_SCAN_AS_USER, 
-				new String[] { PARAM_URL, PARAM_CONTEXT_ID, PARAM_USER_ID, PARAM_MAX_CHILDREN }));
+				new String[] { PARAM_MAX_CHILDREN, PARAM_RECURSE }));
```

Comment:
Sorry, should have told that. The API docs are located in the Messages.properties file.
You can check the keys by running the class JavaAPIGenerator, it will output the missing ones (in this case "spider.api.action.scan" and "spider.api.action.scanAsUser").